Dockerfile-localnet

```diff
@@ -34,11 +50,31 @@ COPY contrib/localnet/zetacored /root/zetacored
 COPY contrib/localnet/tss /root/tss
 
 RUN chmod 755 /root/*.sh
-RUN chmod 700 /root/.ssh
-RUN chmod 600 /root/.ssh/*
 
 WORKDIR /usr/local/bin
-ENV SHELL /bin/sh
 EXPOSE 22
 
-ENTRYPOINT ["/usr/sbin/sshd", "-D"]
+FROM base-runtime AS latest-runtime
+
+COPY --from=cosmovisor-build /go/bin/cosmovisor /usr/local/bin
+COPY --from=latest-build /go/bin/zetacored /go/bin/zetaclientd /go/bin/zetaclientd-supervisor /go/bin/zetae2e /usr/local/bin
+
+# optional old version build
+# use --target latest-runtime to skip
+# 
+# TODO: just download binaries from github release now that we're using glibc
+# we can't do this right now since we do not have a v16 release candidate
```

Comment:
I think it would be great if we can create issues for all TODOs and link those in comment

Reply:
done